### DISCUSSION TITLE
Questions about features

### DISCUSSION THREAD

**Discussion by ovizii** [WillKirkmanM/music#149] [Q&A] [answered]
Hi there,

I just found this project and was wondering about the features since the documentation is sparse.

Is this only usable via its web interface? No clients? No compatibility with other solutions like subsonic or others?

**Comment by WillKirkmanM** [comment, marked as answer]
Hello and Welcome!

Thank you for taking a look at the project.
We have desktop and mobile clients available; desktop through either the Electron or Tauri flavour and mobile through the PWA. The Subsonic Compatibility is something that was in questioning during development,  I did not want this project to be a wrapper for Subsonic and have then opted to implement the features manually. Most, if not all, of the features listed on [Subsonic's Website](https://www.subsonic.org/pages/features.jsp) are implemented already. 

Here are some listed for clarity:
 - Synced Lyrics
 - Audio Tagging
 - MusicBrainz & TADB Metadata Providers
 - CoverArtArchive Integration
 - Playlists
 - Social (Friend Adding)
 - Listen History 
 - Genre Categorisation
and a lot more...

**Reply by ovizii** [reply]
Not sure where my reply went, I am sure I posted it so lets try again. 

Thanks for your reply and explaining. I am not particularly interested in subsonic, it was simply the first name that I remembered.
